### PR TITLE
refactor VolunteerViewModal-css

### LINKED ISSUE #2893: Refactor CSS in `src/screens/EventVolunteers/Volunteers/VolunteerViewModal.tsx`
# Refactor CSS in this file

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

This file contains embedded CSS references that should be completely moved to:
- `src/style/app.module.css`

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is for this file to use this new design pattern. This means that:
1. All embedded CSS in this file must be merged into the global CSS file using pre-existing CSS.
1. All embedded CSS in this file must be removed

### PR DESCRIPTION
**Pull request by Dhiren-Mhatre**
# Pull Request Title  
Refactor CSS in `src/screens/EventVolunteers/Volunteers/VolunteerViewModal.tsx` (#2893)

## What kind of change does this PR introduce?  
Refactoring  

## Issue Number  
Fixes #2893  

## Did you add tests for your changes?  
No. (Refactoring does not require new tests as functionality remains unchanged.)  
 
## Snapshots/Videos  
Screenshot from 2024-12-30 14-19-29

## If relevant, did you update the documentation?  
N/A  

## Summary  
This PR addresses issue #2893, which involves refactoring the CSS within the `src/screens/EventVolunteers/Volunteers/VolunteerViewModal.tsx` file.  

### Motivation for the change:  
- To enhance the UI/UX of Talawa-Admin for colorblind users.  
- To streamline all CSS into a single global file (`src/style/app.module.css`).  

### Tasks completed:  
1. Moved all embedded CSS in `VolunteerViewModal.tsx` to the global CSS file (`src/style/app.module.css`).  
2. Removed all embedded CSS from `VolunteerViewModal.tsx`.  

### Reference  
Foundational work for this change was initiated in PR #2466 (Chore/css changes).  

## Does this PR introduce a breaking change?  
No.  

## Other information  
- The refactored CSS adheres to the design pattern established in `src/style/app.module.css`.  
- Improved maintainability and consistency of CSS across the application.  

## Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?  
Yes.  

## Summary by CodeRabbit

- **Style**
	- Updated styling for the Volunteer View Modal with new CSS classes.
	- Enhanced visual consistency for modal elements, status indicators, and table components.
	- Improved styling for images, headers, and form groups.

- **Documentation**
	- Updated README with new logo and installation instructions.
	- Added more context for developer setup.

- **Chores**
	- Removed unit tests for the TagNode component.
	- Deleted mock data file related to user tags.
	- Updated the TagNode component by removing the checkbox ID attribute.

### CODE REVIEW

File: src/style/app.module.css [hunk 1]
```diff
@@ -4052,3 +4052,66 @@ button[data-testid='createPostBtn'] {
...
+  margin: 0 auto;
+  margin-bottom: 0.5rem;
+}
+
+.statusIcon {
+  margin-right: 0.5rem;
+}
+
+.acceptedStatus {
+  color: green;
```
Comment (palisadoes) [inline src/style/app.module.css:4089]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466 
1. Use a variable instead

File: src/style/app.module.css [hunk 2]
```diff
@@ -4052,3 +4052,66 @@ button[data-testid='createPostBtn'] {
...
+  -webkit-text-fill-color: green;
```
Comment (palisadoes) [inline src/style/app.module.css:4090]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466 
1. Use a variable instead

File: src/style/app.module.css [hunk 3]
```diff
@@ -4052,3 +4052,66 @@ button[data-testid='createPostBtn'] {
...
+}
+
+.pendingStatus {
+  color: #ed6c02;
```
Comment (palisadoes) [inline src/style/app.module.css:4094]: Make this a variable. There should be pre-existing colors that closely match this

File: src/style/app.module.css [hunk 4]
```diff
@@ -4052,3 +4052,66 @@ button[data-testid='createPostBtn'] {
...
+  -webkit-text-fill-color: #ed6c02;
```
Comment (palisadoes) [inline src/style/app.module.css:4095]: Make this a variable. There should be pre-existing colors that closely match this

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comments

**Comment by palisadoes** [comment]
Please fix the conflicting file

**Comment by palisadoes** [comment]
Please make coderabbit.ai approve your changes

**Comment by Dhiren-Mhatre** [comment]
Shall I close this pull request since I didn't fetch the latest changes from upstream/develop-postgres before making my latest changes, and therefore my commit shows that I have changed files which I didn't?

**Comment by palisadoes** [comment]
No, just merge with the latest upstream, commit and push to your origin. The PR will automatically be updated

**Comment by Dhiren-Mhatre** [comment]
Thank you for the guidance! I have merged the latest changes from upstream/develop-postgres, committed, and pushed to my branch. The PR is now updated and working as expected.

**Comment by Dhiren-Mhatre** [comment]
I followed the steps to merge with the latest upstream, committed, and pushed to my origin, but it's still showing extra files that I didn't change or is it okay to proceed with the merge in this case?

**Comment by palisadoes** [comment]
Please reopen. I'm not sure what is wrong. Submit only the files for the issue the next time.
Closing

**Pull request closed without merging**